components/odh-notebook-controller/config/base/params.env

```
@@ -0,0 +1 @@
+odh-notebook-controller-image=quay.io/opendatahub/odh-notebook-controller:1.7-35b81f5
```

same as above, maybe this should be
```suggestion
odh-notebook-controller-image=quay.io/opendatahub/odh-notebook-controller:stable
```